test/acceptance/features/convert-batch-to-transaction.feature

```diff
@@ -0,0 +1,11 @@
+Feature:
+  As a payment service
+  I want to validate and extract the payment requests from batch files and convert them to transactions
+  so that the transactions can be processed in the Payment Service
+
+
+  Scenario: Service will produce a Payment Request
+    Given a batch file is received
+    When the file is processed
+    Then a Payment Request is generated
+
```

Comment:
File permissions of this file are too strict which means no one but the owner can access the file.  I wasn't able to access your branch because of this.